cvc5_pythonic_api/cvc5_pythonic.py

```diff
@@ -767,6 +766,10 @@ def _to_sort_ref(s, ctx):
     """
     if debugging():
         instance_check(s, pc.Sort)
+    if s.isString():
+        return StringSortRef(s, ctx)
+    if s.isSequence():
+        return SeqSortRef(s, ctx)
```

Comment:
I see below that `isSequence` is defined for terms, not sorts. Did you mean to define it for sorts? Here, it is used on a sort.

The same applies to `isString`.